api/src/competitor/competitor.entity.ts

```diff
@@ -0,0 +1,63 @@
+import "reflect-metadata";
+import {
+  BaseEntity,
+  Entity,
+  Column,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from "typeorm";
+import { Field, ObjectType } from "type-graphql";
+import { Profession } from "../profession/profession.entity";
+import { Image } from "../image/image.entity";
+import { ModifierAssignment } from "../modifier_assignment/modifierAssignment.entity";
+import { Combat } from "../combat/combat.entity";
+
+@ObjectType()
+@Entity()
+export class Competitor extends BaseEntity {
+  @Field()
+  @PrimaryGeneratedColumn("uuid")
+  id: string;
+
+  @Field()
+  @Column()
+  name: string;
+
+  @Field()
+  @Column()
+  status: string;
+
+  @Field()
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @Field()
+  @UpdateDateColumn()
+  updatedAt: Date;
+
+  @Field(() => Profession)
+  @ManyToOne(() => Profession, (profession) => profession)
+  profession: Profession;
+
+  @Field(() => Image)
+  @ManyToOne(() => Image, (id_avatar_image) => id_avatar_image.id)
```

Comment:
change the underscord to camelcase for id_avatar_image_.id

Reply:
done